android/src/main/java/com/mobilejazz/harmony/kotlin/android/helpers/LocalizedStrings.kt

```diff
@@ -11,9 +12,9 @@ class LocalizedStrings
 
   private val ctx by WeakRef(context)
```

Comment:
I don't see the benefits of using a WeakRef here because of:
- It's going to be a singleton class.
- The context provided must be the Application Context. 

Using a WeakRef could end up with weird behaviours with null references. It's not always a good thing to use.

Reply:
Applied changes